src/main/cli/cli.c

```diff
@@ -4315,6 +4320,15 @@ STATIC_UNIT_TESTED void cliSet(char *cmdline)
                             *data = (int16_t)atoi((const char*) valPtr);
                         }
 
+                        break;
+                    case VAR_UINT32:
+                        {
+                            // fetch data pointer
+                            uint32_t *data = (uint32_t *)cliGetValuePointer(val) + i;
+                            // store value
+                            *data = (uint32_t)atoi((const char*) valPtr);
```

Comment:
`strtoul` may be preferred (but atoi will work correctly)

Reply:
Both sections were just copy/pasted from the `uint16` existing code and the types updated.

Reply:
It works fine for 16 bit, but uint32 will overflow to sign bit of int32

Reply:
Changed to the `%u` string format type and switched from `atoi()` to `strtoul()`. `atoi()` was limiting to `int32` range.